test/integration/common/Paths.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package com.vaticle.typedb.studio.test.integration.common
+
+import java.io.File
+
+object Paths {
+    val SAMPLE_DATA_PATH = File("test/data/sample_file_structure").absolutePath
```

Comment:
Without context, it's not entirely clear what these are used for, or what the distinction is between `TQL_DATA_PATH` and `SAMPLE_DATA_PATH`.

What do we use them for? Is there a clearer way of communicating it?

The file names are a bit off as well. "Query file name" is a little odd without context, because it's difficult to understand why all query files should have this name. (I assume that all of our current tests only create at most 1 query file.)

I'd therefore rename each one to look like `DEFAULT_QUERY_FILE_NAME`, and also, not sure why the file names themselves have `_string` in them - all query files are representations of strings.

Reply:
Our Paths object now looks like: 
```kt
object Paths {
    val SAMPLE_FILE_STRUCTURE_PATH = File("test/integration/data/sample_file_structure").absolutePath
    val SAMPLE_GITHUB_DATA_PATH = File("test/integration/data/sample_github_data").absolutePath

    const val GITHUB_QUERY_FILE_NAME = "query.tql"
    const val GITHUB_DATA_FILE_NAME = "data.tql"
    const val GITHUB_SCHEMA_FILE_NAME = "schema.tql"
}
```
This path layout looks much cleaner and it is more obvious what each file is for.

Regarding the location of `Paths` itself, I believe this is correct - on a practical level, we use all of the values defined here in at least two separate tests. On a modelling level, looking at the package structure and a description of what `Paths` does:

`//test/integration/common/Paths`
Paths contains the common paths used across studio integration tests. 

These definitions line up to me, so the structure seems solid. While it is a little thin, it doesn't belong anywhere else and dissolving it into each of the tests would be too aggressive and introduce more opportunity for error in my opinion.

Reply:
Summary of an in-person conversation:

We agree that allowing the naming of datasets (e.g. `GITHUB`) is appropriate, but most of `Paths` concerns this one dataset that we currently have. We didn't come to a final design during this conversation, but we want to create a scalable model with clear deliniation between different datasets rather than simply bundling a bunch of strings under the existing `Paths` object.

Reply:
Rather than trying to write an interface that encapsulates the shape of all possible future datasets, instead we've moved `Paths` to the data package and included objects within it that define the shape of the datasets themselves. 

By focusing on the experience of reading the tests and developing new ones, we have readable paths that make sense in the context of the tests.